fastai/transforms.py

```diff
@@ -29,10 +31,13 @@ def stretch_cv(x,sr,sc,interpolation=cv2.INTER_AREA):
     return x[cr:r+cr, cc:c+cc]
 
 def dihedral(x, dih):
+    '''performs any of 8 90 rotations or flips for image x.
+    '''
     x = np.rot90(x, dih%4)
     return x if dih<4 else np.fliplr(x)
 
 def lighting(im, b, c):
+    ''' adjusts image's balance and contrast'''
```

Comment:
This should be more specific. Might be easier to add the docstrings *after* the asciidoc sometimes, since for the asciidoc you can do sample images etc -  which will mean you'll have more info about the details.

Also, note that for fastai.transforms a lot of the methods just support one transform (e.g. `lighting` supports `RandomLighting`) - so maybe just add the doc to the transform class, and in the method say 'implementation for RandomLighting' or something